flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/stream/table/ColumnFunctionsTest.scala

```diff
@@ -323,40 +323,6 @@ class ColumnFunctionsTest extends TableTestBase {
 
     verifyAll(tab1, tab2, expected)
   }
-
-  @Test
-  def testRenameColumns(): Unit = {
-    val t = util.addTable[(Double, Long, String)]('a, 'b, 'c)
-
-    val tab1 = t.renameColumns(withColumns('a) as 'd).select("d, b")
-    val tab2 = t.renameColumns("withColumns(a) as d").select('d, 'b)
-
-    val expected =
-      unaryNode(
-        "DataStreamCalc",
-        streamTableNode(t),
-        term("select", "a AS d", "b")
-      )
-
-    verifyAll(tab1, tab2, expected)
-  }
-
-  @Test
-  def testDropColumns(): Unit = {
```

Comment:
Why did you drop this test? I think it is a valid test to verify that we can drop a range of columns.

Reply:
Very good point. It was more of an accident.